src/test/java/uk/gov/pay/adminusers/persistence/entity/ServiceEntityTest.java

```diff
@@ -13,28 +13,10 @@
 
 public class ServiceEntityTest {
 
-    @Test
-    public void addOrUpdateServiceName_shouldUpdateNameWhenAddingEnName() {
```

Comment:
Because that’s the functionality we’re removing — updating the English-language service name no longer changes the `name` property on the `ServiceEntity` test, only the appropriate `ServiceNameEntity` (which will be already referenced from the `ServiceEntity`).

Reply:
sure, but aren't the following tests still valid? ```        assertThat(serviceEntity.getServiceNames().size(), is(1));	
        assertThat(serviceEntity.getServiceNames().get(SupportedLanguage.ENGLISH).getName(), is("newest-en-name"));```

Reply:
That’s tested at a higher level in `ServiceResourceUpdateTest` and `ServiceResourceUpdateNameTest`.

I can add a unit test for the generic functionality of updating `ServiceNameEntity`s on `ServiceEntity` if you like?

Reply:
New unit tests added.